components/eam/src/physics/crm/crm_history.F90

```diff
@@ -96,6 +97,27 @@ subroutine crm_history_init(species_class)
    call phys_getopts(use_ECPP_out = use_ECPP)
    call phys_getopts(MMF_microphysics_scheme_out = MMF_microphysics_scheme)
 
+   ! ! fields that are normally added by cloud_diagnostics_init()
```

Comment:
Prefer not to see commented out code added.

Reply:
I'll delete them.